teleduck/tests/test_lazy_catalog.py

```
@@ -0,0 +1,105 @@
+    from teleduck.server import run_server
+    run_server(db_file, port, read_only=False)
+
+
+class DuckDbLazyCatalogTest(unittest.TestCase):
+    """Proves the catalog is lazy: a table created after the server started is
+    visible in pg_catalog without any re-registration."""
+
+    @classmethod
+    def setUpClass(cls):
+        cls.port = 55462
+        fd, cls.db_file = tempfile.mkstemp(suffix=".db")
```

Ruff (0.15.15)

[warning] Unpacked variable `fd` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)